betfair/betfair.py

```diff
@@ -97,7 +142,42 @@ def keep_alive(self):
         :raises: BetfairAuthError
 
         """
-        self.make_auth_request('keepAlive')
+        if self.login_is_interactive is True:
```

Comment:
Same comment as above--how about making this request in python, and using the `requests` helpers we already have?